orttraining/orttraining/eager/ort_backends.cpp

```diff
@@ -8,6 +8,7 @@
 #include "ort_backends.h"
 #include "ort_log.h"
 #include "core/platform/env.h"
+#include "orttraining/python/orttraining_pybind_common.h"
```

Comment:
why include the header here? i didn't see anything used in this file.

Reply:
This is for using GetORTBackendsManager() which is declared in the .h file.